packages/lesswrong/components/posts/dialogues/DialogueEditorFeedback.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Dispatch, SetStateAction, useState } from 'react';
+import { getSiteUrl, registerComponent } from '../../../lib/vulcan-lib';
+import Button from '@material-ui/core/Button';
+import { useCurrentUser } from '../../common/withUser';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    padding: 20
+  },
+  button: {
+    fontSize: 16,
+    textTransform: 'none',
+    backgroundColor: theme.palette.buttons.startReadingButtonBackground,
+  },
+  feedbackRow: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  }
+});
+
+type clickState = 'unclicked' | 'clickedAndItWorked' | 'clickedAndItDidntWork'
+
+const feedbackButtonClicked = async (user: UsersCurrent | null, post: PostsEdit, setBeenClicked: Dispatch<SetStateAction<clickState>>) => {
+  const data = { title: post.title,
+    id: post._id,
+    url: getSiteUrl() + "posts/" + post._id,
+    coauthors: post.coauthors.map(c => c.displayName).join(', '),
+    author: post.user?.displayName,
+    requester: user?.displayName
+  }
+  const response = await fetch('https://hooks.slack.com/triggers/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', {
```

Comment:
Is this deliberately using a different mechanism from the current "request feedback" mechanism we have for sending posts to Justis?  (Notably, that one didn't seem to work when we tried to use it for lighthaven.space, but...)

Reply:
Hrm well we could do it but I don't check intercom that often and am not yet sure that Justis will take these on